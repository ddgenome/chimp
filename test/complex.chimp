## complex catalytic mechanism, but don't use the kmc lattice
mechanism "complex.mech"
## parameter input
parameter "complex.par"
## complex task
task "complex.task"
