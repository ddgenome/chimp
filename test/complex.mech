# complex (somewhat) catalytic mechansim
A + 2 @ <- k(Ades) -> k(Aads) @@A;
B + @ <- k(Bdes) -> k(Bads) @B;
@@A + @B + @ <- k(srr) -> k(srf) @C + @@@D;
@@@D <- k(Dcom) -> k(Ddis) 3 @E;
@C <- k(Cads) -> k(Cdes) C + @;
@E <- k(Eads) -> k(Edes) E + @;
