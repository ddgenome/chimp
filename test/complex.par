# complex parameters
Aads	7.0e3
Ades	6.2e1
Bads	1.0e1
Bdes	4.7e2
srf	5.0e5
srr	3.0e3
Ddis	1.0e2
Dcom	1.0e1
Cdes	8.0e1
Cads	1.0e0
Edes	7.4e1
Eads	5.0e-1
