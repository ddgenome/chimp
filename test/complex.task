# -*- text -*-
# test task input
begin model complex
  output "complex.out"
  begin integrator kmc
    scale 1.0e16
    begin state
      begin quantity
	p[A] = 1.0e5		# Pa
	p[B] = 1.0e5		# Pa
      end quantity
      begin output
	(1.0e-1 5.0e-1 1.0e-1)
      end output
      begin reactor batch
	temperature 5.0e2	# K
	pressure 2.0e5		# Pa
	volume 1.0e-3		# m^3
	sites 1.0e20
	rate_numerator molecules
	rate_denominator sites
	fluid_quantity pressure
      end reactor
    end state
  end integrator
end model
