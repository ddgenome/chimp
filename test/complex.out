# complex
# x	@	@@@D	@@A	@B	@C	@E	A	B	C	E	steps
0.000000e+00	1.000000e+00	0.000000e+00	0.000000e+00	0.000000e+00	0.000000e+00	0.000000e+00	1.000000e+05	1.000000e+05	0.000000e+00	0.000000e+00	0
1.000034e-01	2.000000e-04	2.810000e-02	2.295000e-01	3.162000e-01	3.380000e-02	1.065000e-01	9.964310e+04	9.958325e+04	1.751363e+02	4.636937e+02	94930
2.000006e-01	2.000000e-04	2.670000e-02	2.231000e-01	3.211000e-01	3.530000e-02	1.171000e-01	9.945450e+04	9.938685e+04	3.671167e+02	1.038323e+03	187138
3.000015e-01	1.000000e-04	2.900000e-02	2.235000e-01	3.192000e-01	3.410000e-02	1.126000e-01	9.926570e+04	9.919963e+04	5.564738e+02	1.602252e+03	278446
4.000014e-01	1.000000e-04	2.730000e-02	2.273000e-01	3.199000e-01	3.420000e-02	1.093000e-01	9.907689e+04	9.901297e+04	7.425864e+02	2.166596e+03	370247
